docs/examples/driver_examples/Qcodes example with Ithaco.ipynb

```diff
@@ -158,239 +238,561 @@
+       "   'instrument': 'qcodes.instrument_drivers.ithaco.Ithaco_1211.Ithaco_1211',\n",
+       "   'instrument_name': 'camp',\n",
+       "   'name': 'risetime',\n",
+       "   'inter_delay': 0,\n",
+       "   'unit': 'msec',\n",
+       "   'vals': '<Enum: {0.03, 1, 0.1, 0.3, 3, 100, 1000, 10, 0.01, 300, 30}>',\n",
+       "   'post_delay': 0,\n",
+       "   'label': 'Rise Time'}},\n",
+       " 'name': 'camp'}"
       ]
      },
-     "execution_count": 16,
+     "execution_count": 11,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "plotQ"
+    "camp.snapshot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up qcodes database and performing a simple measurement"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we want to feed our Itacho using our Keithley sourcemeter and run a simple qcodes measurement to see the output of the swept values passing through Itacho (for knowing how our sourcemeter works, you can refer to its qcodes notebook in this link https://qcodes.github.io/Qcodes/examples/driver_examples/Qcodes%20example%20with%20Keithley%202600.html):"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We initilize a database named \"itacho_db.db\" and save our measured data there. Then, we create an experiment named \"Itacho_measurement\" or load it if is already created:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "initialise_or_create_database_at('itacho_db.db')\n",
+    "exp = load_or_create_experiment('Itacho_measurement', sample_name='no sample')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "{'current': DataArray[10]: current\n",
-       " array([ -4.98941000e-04,  -3.99618000e-04,  -3.00370000e-04,\n",
-       "         -2.01084000e-04,  -1.01763000e-04,  -2.42637000e-06,\n",
-       "          9.69014000e-05,   1.96216000e-04,   2.95535000e-04,\n",
-       "          3.94851000e-04]), 'volt': DataArray[10]: volt\n",
-       " array([ 4.98941  ,  3.99618  ,  3.0037   ,  2.01084  ,  1.01763  ,\n",
-       "         0.0242637, -0.969014 , -1.96216  , -2.95535  , -3.94851  ]), 'volt_set': DataArray[10]: volt_set\n",
-       " array([-5., -4., -3., -2., -1.,  0.,  1.,  2.,  3.,  4.])}"
-      ]
-     },
-     "execution_count": 9,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
    "source": [
-    "data.sync()\n",
-    "data.arrays"
+    "Now, we would like to use qcodes do1d function to sweep over current values\n",
+    "generated by our sourcemeter and measure the current feeding to Itacho and converted voltage values by this instrument. qcodes Measurement can be also used for the same reason, instead.\n",
```

Comment:
This sounds a bit confusing to me. We are not actually measureing the current. Rather we are measuring the voltage on the dmm and converting it to current which we can do because we know the scaling factor (1e-4 A/V)

Reply:
Actually, it is a bit confusing in the driver for me. The docstring of the CurrentParameter is saying: "Current measurement via an Ithaco preamp and a measured voltage". But, it is assigning voltage as a meaured_param in the class. But, I think you are right. The target is to converting cuur to volt and amplifying it.

Reply:
Feel free to also improve the docstring of the instrument :)